sdk/core/Azure.Core/src/ClientOptions.cs

```diff
@@ -99,6 +102,11 @@ public HttpPipelineTransport Transport
         /// </summary>
         public HttpPipelinePolicy? RetryPolicy { get; set; }
 
+        /// <summary>
+        /// Gets the client options for prototol methods.
+        /// </summary>
+        public ProtocolOptions Protocol { get; }
```

Comment:
I think the property name is too general.

Reply:
ProtocolMethods?